extensions/wikia/Search/classes/ResultSet/CombinedMediaResultSet.php

```diff
@@ -3,32 +3,37 @@
  * Class definition for \Wikia\Search\ResultSet\CombinedMediaResultSet
  */
 namespace Wikia\Search\ResultSet;
+
 use Wikia\Search\Result;
 
 /**
  * Result Set for CombinedMedia Select Service
+ *
  * @package Search
  * @subpackage ResultSet
  */
-class CombinedMediaResultSet extends Base
-{
+class CombinedMediaResultSet extends Base {
 	private $titleSet = [];
 
 	/**
 	 * Override to filter out videos with the same title.
+	 *
 	 * @param Result $result
+	 *
 	 * @return $this
 	 */
-	protected function addResult(Result $result) {
-		if( ( sizeof( $this->titleSet ) >= $this->searchConfig->getLimit() ) ||
-			isset($this->titleSet[$result->getTitle()]) ) {
+	protected function addResult( Result $result ) {
+		if ( ( sizeof( $this->titleSet ) >= $this->searchConfig->getLimit() ) ||
+			isset( $this->titleSet[$result->getTitle()] )
+		) {
 			return $this;
-		} else if ( wfFindFile( \Title::newFromText($result->getTitle(), NS_FILE) ) === false) {
+		} else if ( wfFindFile( \Title::newFromText( $result->getTitle(), NS_FILE ) ) === false ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This branch's code block is the same as the block for the branch on line 26. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1871)